rasa/core/channels/facebook.py

```diff
@@ -141,7 +143,21 @@ def send(self, recipient_id: Text, element: Any) -> None:
         # this is a bit hacky, but the client doesn't have a proper API to
         # send messages but instead expects the incoming sender to be present
         # which we don't have as it is stored in the input channel.
-        self.messenger_client.send(element.to_dict(), recipient_id, "RESPONSE")
+        
+        elementlenght = element.to_dict()
+       
+        if 'text' in elementlenght:
+            messagelength = len(elementlenght['text'])
+            delay = messagelength * fb_delay
```

Comment:
Why do you multiply `messagelength * fb_delay`? Can you please add a comment.

Reply:
I suppose to make it more realistic as if it's a very long message, it makes sense to type it for longer

Reply:
Yeah, sounds good. I would just add a comment here.

Reply:
Hi @tabergma 

I added a comment in that line to explain the delay